spyder/plugins/findinfiles/widgets.py

```diff
@@ -1046,8 +1047,9 @@ def update_actions(self):
             w1 = widget.fontMetrics().width(stop_text)
             w2 = widget.fontMetrics().width(search_text)
             # Ensure the search/stop button has the same size independent on
-            # the length of the words.
-            width = self.get_options_menu_button().width() + max([w1, w2]) + 2
+            # the length of the words. The 10 additional pixels account for 
+            # operating system spacing differences
+            width = self.get_options_menu_button().width() + max([w1, w2]) + 10

```

Comment:
This ensure the start/stop button on Find in Files keeps the same size, avoiding some jumps of the items (left/right)

Reply:
Maybe the `10` should go to a constant?

Reply:
It is only used there. Could you elaborate on how this improves ... something  @dalthviz ?

Reply:
Just for readability and if needed to calculate something else in the future but mainly just for style proposes at this moment

Reply:
Fixed.